config/mongo.go

```diff
@@ -57,3 +63,43 @@ func LoadMongoDBFromEnv() *MongoDB {
 	}
 	return &mongo
 }
+
+func (m *MongoDB) mustMode(s *mgo.Session) {
+	if m.Mode == "" {
```

Comment:
Good point. I only see a need for it if we want an error logged for cases where a non existent mode / misspelling is supplied in the config, which could be handled in the default case.